### ISSUE TITLE
The load_data() functions for Citeseer and Pubmed

### ISSUE THREAD

**Issue by Janet180** [CastalZhou/graphsr#1]
Hi, I'm interested in your paper. I noticed that the load_data() functions of the datasets Citeseer and Pubmed are not reported. 

I am aware that the samples of the training set, valid set, and test set are extracted from rand_indices[:1500], rand_indices[1500:2000], and list(rand_indices[2000:]) in the data_spilit() function of the Cora dataset. 

If it's not too much to ask, I was wondering if you select samples for the training sets, valid sets, and test sets of Citeseer and Pubmed using the same index. In other words, 'rand_indices[:1500], rand_indices[1500:2000], and list(rand_indices[2000:])'.

Thanks!
